app/models/ability.rb

```diff
@@ -0,0 +1,40 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the passed in user here. For example:
+    #
+    user ||= User.new # guest user (not logged in)
+    if user.is?('superadmin')
```

Comment:
ach so, ich glaube, die datei ability.rb gehört nicht in den Branch... Hatte ich wohl aus Versehen reingemergt.